sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -414,6 +414,13 @@ void gnrc_ndp_rtr_sol_handle(kernel_pid_t iface, gnrc_pktsnip_t *pkt,
 #endif
             delay = timex_set(0, genrand_uint32_range(0, ms));
             vtimer_remove(&if_entry->rtr_adv_timer);
+#ifdef MODULE_GNRC_SIXLOWPAN_ND_ROUTER
+            gnrc_ipv6_nc_t *nc_entry = gnrc_ipv6_nc_get(iface, &ipv6->src);
+            if (nc_entry != NULL) {
+                vtimer_set_msg(&if_entry->rtr_adv_timer, delay, gnrc_ipv6_pid,
+                        GNRC_NDP_MSG_RTR_ADV_SIXLOWPAN_DELAY, nc_entry);
+            }
+#else
```

Comment:
I haven't checked - is a 6LBR supposed to send router advertisements on its 6lo interface periodically?